Libraries/CO.CDP.AwsServices.Tests/AwsFileManagerTest.cs

```diff
@@ -65,32 +66,19 @@ public async Task UploadFile_UploadsFileUsingPutObject_WhenFileIsSmall()
     {
         using var stream = new MemoryStream(new byte[10]);
         var filename = "smallfile.txt";
+        await _s3Client.PutBucketAsync(new PutBucketRequest { BucketName = StagingBucket });
 
         await _fileManager.UploadFile(stream, filename);
-
-        _mockS3Client.Verify(s3 => s3.PutObjectAsync(It.Is<PutObjectRequest>(req =>
-            req.BucketName == StagingBucket &&
-            req.Key == filename &&
-            req.InputStream == stream &&
-            req.ContentType == "application/octet-stream"
-        ), default), Times.Once);
     }
 
     [Fact]
     public async Task UploadFile_UploadsFileUsingTransferUtility_WhenFileIsLarge()
     {
         using var stream = new MemoryStream(new byte[104857601]);
         var filename = "largefile.txt";
+        await _s3Client.PutBucketAsync(new PutBucketRequest { BucketName = StagingBucket });
 
         await _fileManager.UploadFile(stream, filename);
-
-        _mockS3Client.Verify(s3 => s3.PutObjectAsync(It.IsAny<PutObjectRequest>(), default), Times.Never);
-        _mockTransferUtility.Verify(s3 => s3.UploadAsync(It.Is<Amazon.S3.Transfer.TransferUtilityUploadRequest>(req =>
-            req.BucketName == StagingBucket &&
-            req.Key == filename &&
-            req.InputStream == stream &&
-            req.ContentType == "application/octet-stream"
-        ), default), Times.Once);
```

Comment:
Is there a way to assert if the file was uploaded? We're left with tests that make no assertions.

Reply:
Perhaps, inspecting that `await _fileManager.UploadFile(stream, filename);` result is successful?

Reply:
Above applies to other tests .

Reply:
Very valid point. there is code to validate it now.